output-html.inc

```diff
@@ -1,5 +1,40 @@
 <?php
 
+function check_for_gpg_fingerprint_on_mozilla_keyserver($gpg_key) {
+  $return_bool = false;
+  if (is_callable('curl_init')) {
+    $ch = curl_init();
+    $search_url = 'http://gpg.mozilla.org/pks/lookup?search=0x' . $gpg_key . '&op=vindex';
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+    curl_setopt($ch,CURLOPT_URL,$search_url);
+    $output=curl_exec($ch);
+    $return_code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+    if (isset($return_code)) {
+      if ($return_code == "404"){
+        $return_bool = false;
+      } elseif ($return_code == "200"){
+        $return_bool = true;
+      }
+    }
+    curl_close($ch);
+  }
+  return $return_bool;
+
+}
+
+function output_pgpFingerprint($s) {
+  $stripped_fingerprint = str_replace(" ", "", $s);
+  $show_link = check_for_gpg_fingerprint_on_mozilla_keyserver($stripped_fingerprint);
```

Comment:
re 1) You're right, but I was going to try to mimic what was done on login.mozilla.com. I'm leaning towards just showing the link and calling it good. There are instructions on login.mozilla.com for getting the key on gpg.mozilla.org

re 2) Yeah, you're absolutely right. This is something that I hadn't considered and to me is the clincher as to why this functionality shouldn't be here. I'll display the link, and if it's a 404, it's a 404.